distributions/standalone/assembly-whitelist.txt

```diff
@@ -14,20 +14,6 @@
 /mule-standalone-${productVersion}/MIGRATION.txt
 
 #
-# /apps
-#
-/mule-standalone-${productVersion}/apps/default
```

Comment:
Should not include /mule-enterprise-standalone-${productVersion}/apps in the list?